acmw-db-app/pages/api/attendance/[pid].js

```diff
@@ -0,0 +1,64 @@
+import pgQuery from '../../../postgres/pg-query.js';
+//idk I wanna do this somehow but need to export it from student? thots?
+//import {createStudent} from '../student/[pid].js'; 
```

Comment:
I like the import better than re-writing the SQL if possible - I think you would probably just have to change the end of it to return the student id

Reply:
having issues since student can only have one export